### ISSUE TITLE
Multi-column in multi-column?

### ISSUE THREAD

**Issue by nucspl** [efemkay/obsidian-modular-css-layout#43]
Just tried the snippet today and it's truly a godsend for customization! But as I playing around with how I could go about laying out my dashboard, it seems that I can't stack `multi-column`s.

<details>
 <summary>A demonstrative callout</summary>

Just throw this in a new Obsidian note.
```
%%> [!multi-line}%% %%this ruins the layout%%
>> [!blank-container]
>>> [!multi-column]
>>>> [!manifest]- ### Manifest
>>>> test
>>>
>>>> [!tasklist]- ### Backlog
>>>> test
>>
>>> [!projects] ### Projects
>>> test
```
</details>

---
### The Issue
 I wanted to have this kind of layout:
![image](https://github.com/efemkay/obsidian-modular-css-layout/assets/80261260/710fc522-78e9-46c4-a478-dc366c006459)

And have this table to the left:
![image](https://github.com/efemkay/obsidian-modular-css-layout/assets/80261260/d4182e52-0280-46b7-b613-9297a6969f1b)

But when adding a `multi-column` callout inside another, it does this instead, and happens regardless of line width:
![image](https://github.com/efemkay/obsidian-modular-css-layout/assets/80261260/0bd2e83b-ffc5-49c7-9aa0-58ebe82bcb53)

**Comment by efemkay** [comment]
@nucspl `[!multi-column]` currently cannot have another `[!multi-column]` (i might change that in future). but for now you can do this trick -- making the `[multi-column]` a float callout

![image](https://github.com/efemkay/obsidian-modular-css-layout/assets/42369515/a4815732-cbf5-450b-a52d-739e68ae7b00)

```markdown
### Standard Table with Float `[!multi-column|float-right-large]` Callout
> [!multi-column|float-right-large]
> 
> > [!NOTE] Title
> > Contents
> 
> > [!NOTE] Title
> > Contents
> 
> > [!NOTE] Title
> > Contents

| Col One | Col two | Col three |
| ------- | ------- | --------- |
| item 1  | item 2  | item 3    |
| item 4  | item 5  | item 6    |

```

**Comment by nucspl** [comment]
I see, and it seems to be a cleaner way of doing it too! Thanks!
![image](https://github.com/efemkay/obsidian-modular-css-layout/assets/80261260/2a82a6f1-ab29-4e4c-81d1-31f2097eacef)

Though, things do break without a readable line length override, and the floats appear first on mobile, so stackable `[!multi-column]`s would still be better.
![image](https://github.com/efemkay/obsidian-modular-css-layout/assets/80261260/ffd73e23-c0ef-4e67-a804-4b35225bfdbf)
![image](https://github.com/efemkay/obsidian-modular-css-layout/assets/80261260/e2687f10-6e85-4dcf-91bb-46608bac4cd0)

**Comment by efemkay** [comment]
enhancement released with 0.9.7
